config/findbugs/excludeFilter.xml

```diff
@@ -65,4 +65,9 @@
         <Field name="updated"/>
         <Bug pattern="UWF_UNWRITTEN_PUBLIC_OR_PROTECTED_FIELD"/>
     </Match>
+    <Match>
+        <Class name="com.pubnub.api.PubNubException"/>
+        <Field name="affectedCall"/>
+        <Bug pattern="URF_UNREAD_FIELD"/>
+    </Match>
```

Comment:
Don't know why it started to be a problem now. I didn't touch it at all yet here it is